content/en/blog/_posts/2022-12-01-scalable-job-tracking-ga/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: blog
+title: "Scalable Job tracking goes GA to support massively parallel batch workloads"
+date: 2022-12-01
+slug: "scalable-job-tracking-ga"
+---
+
+**Authors:** Aldo Culquicondor (Google)
+
+The Kubernetes 1.26 release includes a stable implementation of the Job
+controller that can reliably track a large amount of Jobs with high levels of
+parallelism. SIG Apps have worked on this foundational improvement, under
+the feature gate `JobTrackingWithFinalizers`, since Kubernetes 1.22. After
+multiple iterations and scale verifications, this is now the default
+implementation of the Job controller.
+
+Paired with the Indexed [completion mode](/docs/concepts/workloads/controllers/job/#completion-mode),
+the Job controller can handle massively parallel batch Jobs, supporting up to
+100 thousand concurrent Pods.
```

Comment:
curious if 100,000 concurrent pods have been tested?

Reply:
I tested 10k, which runs in ~200s at 100 QPS. But the API allows up to 100k.